### PR TITLE
Rename ff cmds flags

### PR DESCRIPTION
**Pull request by makalaaneesh**

### CODE REVIEW

File: yb-voyager/cmd/export.go [hunk 1]
```diff
@@ -103,9 +108,43 @@ func registerCommonExportFlags(cmd *cobra.Command) {
...
+}
 
-	cmd.Flags().BoolVar(&startClean, "start-clean", false,
-		"cleans up the project directory for schema or data files depending on the export command")
+func registerTargetDBAsSourceConnFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&source.Host, "target-db-host", "127.0.0.1",
+		"host on which the YugabyteDB server is running")
+
+	cmd.Flags().IntVar(&source.Port, "target-db-port", -1,
+		"port on which the YugabyteDB YSQL API is running")
```
Comment (amit-yb) [inline yb-voyager/cmd/export.go:118]: I hope this works when user doesn't provide `--target-db-port`.

File: yb-voyager/cmd/export.go [hunk 2]
```diff
@@ -284,6 +328,14 @@ func validateSourcePassword(cmd *cobra.Command) {
 	}
 }
 
+func validateTargetDBAsSourcePassword(cmd *cobra.Command) {
+	var err error
+	source.Password, err = getPassword(cmd, "target-db-password", "TARGET_DB_PASSWORD")
+	if err != nil {
+		utils.ErrExit("error in getting source-db-password: %v", err)
```
Comment (amit-yb) [inline yb-voyager/cmd/export.go:335]: Copi-pasta error? `source-db-password`?

File: yb-voyager/cmd/import.go [hunk 3]
```diff
@@ -114,15 +123,52 @@ func registerCommonImportFlags(cmd *cobra.Command) {
...
-	cmd.Flags().BoolVar(&startClean, "start-clean", false,
-		"import schema: delete all existing schema objects \nimport data / import data file: starts a fresh import of data or incremental data load")
+func registerFFDBAsTargetConnFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&tconf.Host, "ff-db-host", "127.0.0.1",
+		"host on which the Fall-forward DB server is running")
 
-	cmd.Flags().BoolVar(&tconf.VerboseMode, "verbose", false,
-		"verbose mode for some extra details during execution of command")
+	cmd.Flags().IntVar(&tconf.Port, "ff-db-port", -1,
+		"port on which the Fall-forward DB YSQL API is running")
```
Comment (priyanshi-yb) [inline yb-voyager/cmd/import.go:134]: `YSQL API`?

File: yb-voyager/cmd/import.go [hunk 4]
```diff
@@ -114,15 +123,52 @@ func registerCommonImportFlags(cmd *cobra.Command) {
...
+
+	cmd.Flags().StringVar(&tconf.OracleHome, "oracle-home", "",
+		"[For Oracle Only] Path to set $ORACLE_HOME environment variable. tnsnames.ora is found in $ORACLE_HOME/network/admin")
+
+	cmd.Flags().StringVar(&tconf.TNSAlias, "oracle-tns-alias", "",
+		"[For Oracle Only] Name of TNS Alias you wish to use to connect to Oracle instance. Refer to documentation to learn more about configuring tnsnames.ora and aliases")
+
+	cmd.Flags().StringVar(&tconf.Schema, "ff-db-schema", "",
+		"schema name in Fall-forward DB (Note: works only for source as Oracle and MySQL, in case of PostgreSQL you can ALTER schema name post import)")
+
```
Comment (priyanshi-yb) [inline yb-voyager/cmd/import.go:156]: I think, no need of MySQL/PG mention.

File: yb-voyager/cmd/export.go (hunk 1 above)
Comment (makalaaneesh) [reply yb-voyager/cmd/export.go:118]: Ah, yes. `setSourceDefaultPort` takes care of YB as well.

File: yb-voyager/cmd/export.go [hunk 5]
```diff
@@ -284,6 +328,14 @@ func validateSourcePassword(cmd *cobra.Command) {
 	}
 }
 
+func validateTargetDBAsSourcePassword(cmd *cobra.Command) {
```
[the remark this answers is not in the document]
Comment (makalaaneesh) [reply yb-voyager/cmd/export.go:331]: Yeah, we need to revisit these functions. They're doing too many things, and validate/get are not apt names. Was discussing this with Priyanshi too :) 

For now, renamed it to `getAndStoreTargetDBPasswordInSourceConf` to be as explicit as possible 😅

File: yb-voyager/cmd/export.go (hunk 2 above)
Comment (makalaaneesh) [reply yb-voyager/cmd/export.go:335]: yes! Fixed, thanks

**Pull request merged**